tools/meteor-services/deploy.js

```diff
@@ -21,6 +21,8 @@ import {
 } from './auth.js';
 import { recordPackages } from './stats.js';
 import { Console } from '../console/console.js';
+import { setTimeout } from 'timers';
+var fiberHelpers = require('../utils/fiber-helpers.js');
```

Comment:
Is there a reason we can't use the more modern `import` instead of `require` here?